src/Marten.TestHelpers/DaemonContextHelper.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Threading.Tasks;
+using JasperFx.Core;
+using Marten.Events.Daemon;
+using Marten.Events.Daemon.HighWater;
+using Marten.Events.Projections;
+using Microsoft.Extensions.Logging;
+
+namespace Marten.TestHelpers;
+
+public class DaemonContextHelper(string schemaName, string connectionString, ILogger<IProjection> logger) : OneOffConfigurationsHelper(schemaName, connectionString)
```

Comment:
**SUGGESTION:** I don't like the `Helper` suffix. I would either name it `DaemonContext` or `ProjectionDaemonRunner`.

Reply:
Will do